Documents/Help/Changelog.md

```diff
@@ -1,5 +1,14 @@
 =======
 
+## 2022-11-xx - Build 2211 - November 2022
```

Comment:
This needs to be added to the Milestone work, Or is it V7 if so then Append Nov 22 to the V7 name

Reply:
Done